### PR TITLE
Start creating step contact information

### PR DESCRIPTION
**Pull request by AhmedQeshta**
## Start creating step contact information

### CODE REVIEW

File: client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
-  StepTwoComponent,
+  ContactInformation,
 } from "./Steps";
 
 export default function BodyStepper() {
   const activeStep = useSelector(({ stepper }) => stepper.activeStep);
 
+  const SignupSchema = Yup.object().shape({
+    city: Yup.string().required("حقل المدينة مطلوب"),
+    phone: Yup.string()
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx:16]: remove schema outside component

File: client/src/Components/RegisterPageComponents/Stepper/Steps/StepThreeComponent.jsx [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import { useDispatch } from "react-redux";
 
+import PropTypes from "prop-types";
+
 import { handleBack } from "../../../../Store/Slices";
 
-export default function StepThreeComponent() {
+export default function StepThreeComponent({ contactForm }) {
+  console.log(contactForm.values);
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/Steps/StepThreeComponent.jsx:9]:
```suggestion
```

File: client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx [hunk 3]
```diff
@@ -1,25 +1,48 @@
...
+      .min(9, "الرقم يجب أن يكون 9 أرقام على الأقل")
+      .max(10, "الرقم يجب أن  لا يكون 10 أرقام")
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx:18]:
```suggestion
     .length(10, message)
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by AFahgan — APPROVED** [review]
Great

**Review by rawaneyad — APPROVED** [review]
Good Job 👍🏼

**Pull request merged**